assets/tests/components/ladderPage.test.tsx

```diff
@@ -525,6 +532,52 @@ describe("LadderPage", () => {
     )
     expect(result.queryByText("Vehicles")).toBeNull()
   })
+
+  test("opens the detour modal", async () => {
+    jest
+      .mocked(getTestGroups)
+      .mockReturnValue([
+        TestGroups.DetourRouteSelection,
+        TestGroups.DetoursPilot,
+        TestGroups.RouteLadderHeaderUpdate,
+      ])
+
+    const mockState = {
+      ...initialState,
+      routeTabs: [
+        routeTabFactory.build({
+          selectedRouteIds: ["1"],
+          ordering: 0,
+          isCurrentTab: true,
+        }),
+      ],
+    }
+    const { container } = render(
+      <StateDispatchProvider state={mockState} dispatch={mockDispatch}>
```

Comment:
suggestion: I don't think we should be accessing a "global" variable in tests, can we replace this?

```suggestion
      <StateDispatchProvider state={mockState} dispatch={jest.fn()}>
```